daemon/cluster/filters.go

```diff
@@ -96,3 +96,21 @@ func newListTasksFilters(filter filters.Args, transformFunc func(filters.Args) e
 
 	return f, nil
 }
+
+func newListSecretsFilters(filter filters.Args) (*swarmapi.ListSecretsRequest_Filters, error) {
+	accepted := map[string]bool{
+		"names": true,
+		"name":  true,
+		"id":    true,
+		"label": true,
```

Comment:
It's not possible to add `label` at the creation of a secret, not sure why this filter should be accepted (it's not gonna filter anything :sweat_smile: — or I'm missing something).